surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,80 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise` and `async function`
+
+(Multiple choice)
+
+  - [ ] Promises
+  - [ ] async/await
+  - [ ] Not writing Promise-based code
+
+## How are you handling rejections today?
+
+When consuming Promises, async functions or thenables, which of the options below do you use to handle rejections?
+
+(Multiple choice)
+
+  - [ ] `.catch()`
+  - [ ] `try / catch` wrapping an `await` operation
+  - [ ] Ignore the rejection
+  - [ ] Not writing Promise-based code
+
+## Do you use `process.on('unhandledRejection')` as a global handler for unhandled rejections?
+ 
+  - [ ] Yes
+  - [ ] No
+
+## Do you use the `--unhandled-rejections` command-line flag as a global handler for unhandled rejections?
+
+  - [ ] Yes
+  - [ ] No
+
+## Are you using Promises in customer facing, production applications?
+
+  - [ ] Yes
+  - [ ] No
+  - [ ] I don't know
+
+## Which of the use cases described below do you spend most of your time on?
```

Comment:
this should probably be multiple choice?

Reply:
Might make sense, although I was trying to capture majority of time spent (and not everything someone works on).